src/AdoNet/Orleans.Persistence.AdoNet/Storage/Provider/AdoNetStorageProvider.cs

```diff
@@ -552,7 +552,8 @@ private ICollection<IStorageDeserializer> ConfigureDeserializers(IProviderConfig
             var deserializers = new List<IStorageDeserializer>();
             if(config.Properties.ContainsKey(UseJsonFormatPropertyName) && @true.Equals(config.Properties[UseJsonFormatPropertyName], StringComparison.OrdinalIgnoreCase))
             {
-                var jsonSettings = OrleansJsonSerializer.UpdateSerializerSettings(OrleansJsonSerializer.GetDefaultSerializerSettings(this.serializationManager, providerRuntime.GrainFactory), config);
+                var typeResolver = providerRuntime.ServiceProvider.GetRequiredService<ITypeResolver>();
```

Comment:
FYI: see https://github.com/dotnet/orleans/issues/3088#issuecomment-362511050 and https://gitter.im/dotnet/orleans?at=5a741d07ce68c3bc7435dee9 to make this more adjustable in the future (not an issue in this PR, I suppose, adding here just for the reference).